microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -0,0 +1,69 @@
+"""
+Testing fixtures (e.g. for CRUD).
+
+"""
+from uuid import uuid4
+
+from marshmallow import fields, Schema
+
+from microcosm_flask.linking import Links, Link
+from microcosm_flask.operations import Operation
+
+
+class Person(object):
```

Comment:
This is a simple example of using the prerequisites for using the CRUD convention. It has:
 - a model (Person)
 - a schema for creating new people
 - a schema for a fully populated person
 - trivial/naive functions for each CRUD operation